Packs/Uptycs/Integrations/Uptycs/Uptycs.py

```diff
@@ -437,11 +437,13 @@ def uptycs_get_alerts_command():
 
     if context is not None:
         for index in range(len(context)):
-            if bool(json.loads(context[index].get('metadata')).get('pid')):
-                context[index]['pid'] = json.loads(
-                    context[index].get('metadata')).get('pid')
-            else:
-                context[index]['pid'] = 'Not applicable or unknown'
+            for key in ['pid', 'PID', 'Process ID']:
+                if bool(json.loads(context[index].get('metadata')).get(key)):
+                    context[index]['pid'] = json.loads(
+                        context[index].get('metadata')).get(key)
+                    break
+                else:
+                    context[index]['pid'] = 'Not applicable or unknown'
```

Comment:
as in python `bool('False') == True`, so if in `bool(metadata.get(key))` you just need to check if it's not None - you can use `if metadata.get(key)` it will check if the value are't `None` or `''`
so please check my suggestion 
```suggestion
            metadata = json.loads(context[index].get('metadata'))
            for key in ['pid', 'PID', 'Process ID']:
                if metadata.get(key):
                    context[index]['pid'] = metadata.get(key)
                    break
                else:
                    context[index]['pid'] = 'Not applicable or unknown'
```